packages/devextreme/testing/testcafe/tests/dataGrid/keyboardNavigation/keyboardNavigation.functional.ts

```diff
@@ -12,6 +12,50 @@ const CLASS = ClassNames;
 fixture.disablePageReloads`Keyboard Navigation - common`
   .page(url(__dirname, '../../container.html'));
 
+test('Next cell should be focused immediately on a single Enter key press if showEditorAlways is enabled in cell mode (T1196539)', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .click(dataGrid.getDataCell(0, 0).element)
+    .typeText(dataGrid.getDataCell(0, 0).element, 'test')
+    .pressKey('enter');
+
+  await takeScreenshot('cell-focus-showEditorAlways-T1196539-1.png', dataGrid.element);
```

Comment:
How about avoid screenshots and use properties like "isEditCell", "isFocused", etc?

Reply:
Hello,

I get rid of other screenshots. I left one screenshot test because there is no property to check if the editor's text is selected or not. The isEditCell property can't be used here because showEditorAlways is enabled and the property is always true. Technically, I could check input selection, but I chose a screenshot test instead :)